components/amilia/actions/get-account/get-account.mjs

```diff
@@ -0,0 +1,35 @@
+import amilia from "../../amilia.app.mjs";
+
+export default {
+  key: "amilia-get-account",
+  name: "Get Account",
+  description: "Get an (client) account in your organization. [See the docs here](https://www.amilia.com/ApiDocs/v3org#GetAnAccount)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    amilia,
+    account: {
+      propDefinition: [
+        amilia,
+        "account",
+      ],
+    },
+    showMedicalInfo: {
+      type: "boolean",
+      label: "Show Medical Info?",
+      description: "Option to also show the persons' medical information.",
+      optional: true,
+    },
+  },
+  async run({ $ }) {
+    const response = await this.amilia.getAccount({
+      $,
+      account: this.account,
+      params: {
+        showMedicalInfo: this.showMedicalInfo,
+      },
+    });
+    $.export("$summary", "Succesfully retrieved account");
```

Comment:
Try to add an identificator to the `export` message, you can do this in the rest too.

Obs: I didn't tested this!

```suggestion
    $.export("$summary", `Succesfully retrieved account with id ${response.Id}`);
```